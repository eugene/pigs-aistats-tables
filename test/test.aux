\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand{\transparent@use}[1]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\bibstyle{unsrtnat}
\providecommand \oddpage@label [2]{}
\@writefile{toc}{\contentsline {section}{\numberline {1}Lipsum}{1}{section.1}\protected@file@percent }
\gdef \LT@i {\LT@entry 
    {5}{102.8681pt}\LT@entry 
    {5}{58.65529pt}\LT@entry 
    {1}{70.24968pt}\LT@entry 
    {1}{104.71349pt}\LT@entry 
    {5}{83.64919pt}\LT@entry 
    {1}{41.71498pt}}
\gdef \LT@ii {\LT@entry 
    {5}{102.8681pt}\LT@entry 
    {1}{36.77174pt}\LT@entry 
    {1}{41.15956pt}\LT@entry 
    {1}{31.162pt}\LT@entry 
    {1}{62.54323pt}\LT@entry 
    {1}{33.24483pt}\LT@entry 
    {1}{27.96834pt}}
\gdef \LT@iii {\LT@entry 
    {5}{102.8681pt}\LT@entry 
    {5}{58.65529pt}\LT@entry 
    {1}{70.24968pt}\LT@entry 
    {1}{104.71349pt}\LT@entry 
    {5}{73.65163pt}\LT@entry 
    {5}{88.64796pt}}
\@writefile{lot}{\contentsline {table}{\numberline {1}{\ignorespaces UCI benchmarks - $\mathcal  {L}$\relax }}{3}{table.1}\protected@file@percent }
\@writefile{lot}{\contentsline {table}{\numberline {2}{\ignorespaces UCI benchmarks - $\qopname  \relax o{log}p(y|x)$\relax }}{3}{table.2}\protected@file@percent }
\gdef \LT@iv {\LT@entry 
    {5}{102.8681pt}\LT@entry 
    {5}{58.65529pt}\LT@entry 
    {5}{58.65529pt}\LT@entry 
    {5}{58.65529pt}\LT@entry 
    {1}{62.54323pt}\LT@entry 
    {5}{78.6504pt}\LT@entry 
    {5}{68.65285pt}}
\gdef \LT@v {\LT@entry 
    {5}{102.8681pt}\LT@entry 
    {5}{55.32277pt}\LT@entry 
    {1}{70.24968pt}\LT@entry 
    {1}{104.71349pt}\LT@entry 
    {1}{73.15175pt}\LT@entry 
    {5}{55.32277pt}}
\gdef \LT@vi {\LT@entry 
    {5}{102.8681pt}\LT@entry 
    {5}{55.32277pt}\LT@entry 
    {5}{55.32277pt}\LT@entry 
    {5}{55.32277pt}\LT@entry 
    {1}{62.54323pt}\LT@entry 
    {5}{55.32277pt}\LT@entry 
    {5}{55.32277pt}}
\@writefile{lot}{\contentsline {table}{\numberline {3}{\ignorespaces UCI benchmarks - RMSE $[y, \mu (x)]$\relax }}{4}{table.3}\protected@file@percent }
\gdef\svg@ink@ver@settings{{\m@ne }{inkscape}{\m@ne }}
\gdef \@abspage@last{4}
