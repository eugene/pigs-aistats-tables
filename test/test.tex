\documentclass[twoside]{article}

\usepackage{aistats2022}
% If your paper is accepted, change the options for the package
% aistats2022 as follows:
%
%\usepackage[accepted]{aistats2022}
%
% This option will print headings for the title of your paper and
% headings for the authors names, plus a copyright note at the end of
% the first column of the first page.

% If you set papersize explicitly, activate the following three lines:
%\special{papersize = 8.5in, 11in}
%\setlength{\pdfpageheight}{11in}
%\setlength{\pdfpagewidth}{8.5in}

% If you use natbib package, activate the following three lines:
\usepackage[round]{natbib}
\renewcommand{\bibname}{References}
\renewcommand{\bibsection}{\subsubsection*{\bibname}}

% If you use BibTeX in apalike style, activate the following line:
%\bibliographystyle{apalike}
\usepackage[utf8]{inputenc} % allow utf-8 input
\usepackage[T1]{fontenc}    % use 8-bit T1 fonts
\usepackage{hyperref}       % hyperlinks
\usepackage{url}            % simple URL typesetting
\usepackage{booktabs}       % professional-quality tables
\usepackage{amsfonts}       % blackboard math symbols
\usepackage{nicefrac}       % compact symbols for 1/2, etc.
\usepackage{microtype}      % microtypography
\usepackage{xcolor}         % colors

% ==============================
% My packages
\usepackage{amsmath}
\usepackage{svg}
\usepackage{pdfpages}
\usepackage{wrapfig}
\usepackage{longtable}
\usepackage{lipsum}
\usepackage[ruled]{algorithm2e}
\usepackage{graphicx,multirow}
\usepackage{xfrac}
\usepackage{subcaption}
\usepackage{csvsimple}
\usepackage{lscape}
\usepackage[normalem]{ulem}




% Unnumber algoright
% \makeatletter
% \newcommand{\RemoveAlgoNumber}{\renewcommand{\fnum@algocf}{\AlCapSty{\AlCapFnt\algorithmcfname}}}
% \newcommand{\RevertAlgoNumber}{\algocf@resetfnum}
% \makeatother

% Colours
\definecolor{navyBlue}{HTML}{030F4F}
\definecolor{primaryRed}{HTML}{990000}
\definecolor{editGreen}{HTML}{40916c}


% My inputs
%\input{setup/math}
\bibliographystyle{unsrtnat}

\begin{document}

\twocolumn[
\aistatstitle{Robust uncertainty estimates with out-of-distribution pseudo-inputs training}

\aistatsauthor{ Pierre Segonne \And Yevgen Zainchkovskyy \And  SÃ¸ren Hauberg}

\aistatsaddress{ Institution 1 \And  Institution 2 \And Institution 3 } ]

\begin{abstract}
  Probabilistic models often use neural networks to control their predictive uncertainty.
  However, when making \textit{out-of-distribution (OOD)} predictions, the often-uncontrollable 
  extrapolation properties of neural networks yield poor uncertainty predictions. Such models then don't \textit{know what they don't know}, which directly limits their robustness w.r.t unexpected inputs.
  To counter this, we propose to explicitly train the uncertainty predictor where we are not given data
  to make it reliable. As one cannot train without data, we provide mechanisms for generating
  \emph{pseudo-inputs} in informative low-density regions of the input space, and show how to leverage these in a practical
  Bayesian framework that casts a prior distribution over the model uncertainty. With a holistic evaluation, we demonstrate that this yields robust and interpretable predictions of uncertainty while retaining
  state-of-the-art performance on diverse tasks such as regression and generative modelling.\looseness=-1
\end{abstract}

% ==========================
% \input{sections/introduction}
% \input{sections/method}
% \section{Experiments}
% \input{sections/regression}
% \input{sections/generative_modelling}
% \input{sections/conclusion}
% ==========================

% ==========================
% DO NOT INCLUDE AT SUBMISSION TIME
% \textbf{\large Acknowledgements.} ...
\section{Lipsum}
\lipsum

%\bibliography{bibliography}

% DO NOT INCLUDE AT SUBMISSION TIME
% \input{appendix}

\clearpage
\onecolumn
% \begin{landscape}

\input{../output/1-table_elbo_not_shifted.tex}
\input{../output/2-table_ELL.tex}
\input{../output/3-table_RMSE.tex}

% \end{landscape}
\end{document}