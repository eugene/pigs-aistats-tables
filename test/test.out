\BOOKMARK [1][-]{section.1}{\376\377\000L\000i\000p\000s\000u\000m}{}% 1
